### PR TITLE
Implement input debounce, also added `readOnly` and `enabled` properties

### PR DESCRIPTION
**Pull request by danvick**
I noticed that we have `futureRebuildDuration` but the logic for it wasn't implemented.

We may want to enable/diable the field or make it read-only

### CODE REVIEW

**Review by theiskaa — CHANGES_REQUESTED** [review]
It works now, but you opened pr from your main branch, so develop had changes, so you need to resolve the conflicts or create PR from ur develop branch to our develop :).

**Comment by danvick** [comment]
Done!

File: pubspec.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   highlightable: ^1.0.5
+  rxdart: ^0.27.7
```
Comment (theiskaa) [inline pubspec.yaml:16]: Why we need this?

**Review by theiskaa — CHANGES_REQUESTED** [review]
Since you resolved conflicts manually there is some unwanted changes.

Can you close and make your changes to develop and open to merge to develop?

**Comment by theiskaa** [comment]
@danvick please move to develop branch, make your changes and then open pr to develop. Current changes 
cannot be merged because its over-changes.

File: pubspec.yaml (hunk 1 above)
Comment (danvick) [reply pubspec.yaml:16]: RxDart is a package for handling streams - that's what I used it for.

**Comment by danvick** [comment]
I'll do that as soon as I can. It may take a day or two though.

File: pubspec.yaml (hunk 1 above)
Comment (theiskaa) [reply pubspec.yaml:16]: Cannot be implemented with default dart streams?

**Comment by theiskaa** [comment]
> I'll do that as soon as I can. It may take a day or two though.

For sure @danvick no problem at all, just wanted to make sure we keep it minimal with no extra dependencies.

**Comment by theiskaa** [comment]
I’m closing this PR as it is outdated and does not meet the requirements requested from us

**Pull request closed without merging**